### ISSUE TITLE
add Hacktoberfest label

### ISSUE THREAD

**Issue by 4ndrej** [kristate/krackinfo#127]
There is annual Digital Ocean + GitHub event for supporting open source development of any kind.
This repo is opensource documentation project.
Adding Hacktoberfest label to project and using this label in issues allows commiters to win prizes from Digital Ocean. More info at https://hacktoberfest.digitalocean.com/#details and https://hacktoberfest.digitalocean.com/#resources

**Comment by kristate** [comment]
I think I added it correctly -- could you check for me, please? Thanks!

**Comment by 4ndrej** [comment]
tl; dr: the label is not there but it's not big problem anyway.

Hello,
I checked it on https://github.com/kristate/krackinfo/labels but there is no hacktoberfest label.

But it seems that it's not relevant for the contest itself. I found out that there does not have to be hacktoberfest label on the issue/pull request to be counted into the Hacktoberfest score anyway. They are counting only pull requests made by registered users.

The label is only for guiding new users to the Hacktoberfest campain.

I also found out that I am not able to apply any labels on any issue / pull request in the repo.

Sorry for spending your time with this.
a.
